src/extension.ts

```diff
@@ -528,6 +529,20 @@ function launchMetals(
       }
     );
 
+    registerCommand("metals.reset-choice-interactive", (_) => {
+      client.sendRequest(ExecuteCommandRequest.type, {
+        command: "reset-choice",
+        arguments: [],
+      });
+    });
+
+    registerCommand("metals.reset-choice", (args) => {
+      client.sendRequest(ExecuteCommandRequest.type, {
+        command: "reset-choice",
+        arguments: args,
+      });
+    });
+
```

Comment:
Shouldn't these also be added under the `commands` in the `package.json`?

Reply:
good catch!
only `metals.reset-choice-interactive` should be added?
as I understand commands in package.json will be added to 'Show all commands' popup?
And those commands must be parameterless?

Will double check

Reply:
Correct. 🤔 Thinking about this I now for myself, since `reset-choice-interactive` is really the only one I will use, I may actually only register `reset-choice` as the name.

Reply:
added :)
![2020-07-10-111555_688x143_scrot](https://user-images.githubusercontent.com/10478402/87138694-1704ca80-c29f-11ea-804d-8e49fc13ee00.png)